book/src/specs/kimchi.md

```diff
@@ -748,7 +748,7 @@ $$E(\mathbb{F}_q): y^2 = x^3 + a x + b$$
 If $P=(x_p, y_p)$ and $Q=(x_q, y_q)$ are two points in the curve $E(\mathbb{F}_q)$, the algorithm we
 represent here computes the operation $2P+Q$ (point doubling and point addition) as $(P+Q)+Q$.
 
-(Remark: point $Q$ has nothing to do with the order $q$ of the field)
+(Notation remark: point $Q$ has nothing to do with the order $q$ of the field)
```

Comment:
note that you can use callouts for these as well: https://github.com/tommilligan/mdbook-admonish

Reply:
Oh this is cute, I will use them instead!